### ISSUE TITLE
New chapter "Extension Publishing"

### ISSUE THREAD

**Issue by twiro** [symphonists/symphony-extensions-network#12]
I took the introduction of github topics as a reason to finally add a second chapter to this repo - it's about recommendations and best practices regarding "**[Extension Publishing](https://github.com/symphonists/symphony-extensions-network/tree/integration#3-extension-publishing)**".

This is only a draft for now and at least the two following points need to be discussed and agreed upon before publishing the chapter to the master-branch.

1. Should the `1.0.0`-syntax be promoted as the official best practice for version-tags of github-releases or should we stick with github's own recommendation of adding a prefixed "v":`v1.0.0`? I'd vote for the first option as this is how Symphony has used version-tags on github since the repo moved there. And not to forget - the prefixed "v" results in missing links in the compatibility-tables on symphonyextension.com... though I guess & hope this could be fixed.
2. Should publishing extensions on [getsymphony.com](http://www.getsymphony.com/download/extensions/) still be recommended? I'm not so sure...

Any other feedback or ideas hot to optimize this chapter are welcome!

**Comment by nitriques** [comment]
1. Screw the v. Both composer and npm does not use it. Bye.
2. No, simply http://symphonyextensions.com/

**Comment by twiro** [comment]
> _ Screw the v. Both composer and npm does not use it. Bye._
> _ No, simply http://symphonyextensions.com/_

Fine - thanks for getting this nailed down quickly!

Finished the chapter and will push to master as soon as I have your OK for the [current draft in the integration-branch](https://github.com/symphonists/symphony-extensions-network/tree/integration#3-extension-publishing).

**Comment by nitriques** [comment]
> Use branches 

I would also advise about a integration branch, which should receive all pull request and contain stable code.

> Nicolas

You might want to consider adding (nitriques) or simply use the username, since I think I might be more recognizable (???)

Everything else LGTM !

**Comment by twiro** [comment]
> _I would also advise about a integration branch, which should receive all pull request and contain stable code._

Good advice! I changed the wording in that section to the following:

_Use **branches** to develop and maintain your extension on github. Your basic setup should contain a **master**-branch (representing the latest stable, tested and documented release) and an **integration**-branch (that is used for ongoing development and incoming pull requests and should cointain stable code too). Experimental and instable stuff should happen in separate feature-branches._

> _You might want to consider adding (nitriques) or simply use the username, since I think I might be more recognizable (???)_

Changed that too!

> _Everything else LGTM !_

Cool - I released the new chapter in the [master-branch](https://github.com/symphonists/symphony-extensions-network#3-extension-publishing) and will try to tackle the task "[Extension Development](https://github.com/symphonists/symphony-extensions-network/issues/13)" next...
